### PR TITLE
[Uid] Remove experimental for UID

### LINKED ISSUE #14890: [Uid] remove "experimental" status
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#40010
| PR author(s) | @nicolas-grekas
| Merged in    | 5.3-dev

### PR DESCRIPTION
**Pull request by OskarStark**
Fix #14890

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Thanks Oskar.

File: notifier.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ the users (e.g. SMS, Slack messages, emails, push notifications, etc.). The
 Notifier component in Symfony is an abstraction on top of all these
 channels. It provides a dynamic way to manage how the messages are sent.
 Get the Notifier installed using:
-
+uid
```
Comment (fabpot) [inline notifier.rst:19]: Is it really needed?

Comment (OskarStark) [reply notifier.rst:19]: Ups ... fix on the way, thanks!

Comment (OskarStark) [reply notifier.rst:19]: I think it was already removed by Javier

Comment (javiereguiluz) [reply notifier.rst:19]: Yes, removed while merging 👍

**Pull request merged**